.github/workflows/extract-translation-source-files.yml

```diff
@@ -214,7 +215,8 @@ jobs:
           cd translations/${{ matrix.repo }}
           # finds the directory containing the english locale usually located in
           # */*/conf/locale/en
-          EN_DIR=$(find . -type d -regex ".+conf\/locale\/en$")
+          # but also exclude any hidden directories that might exist for some reason
+          EN_DIR=$(find . -type d -not -path '*/.*' -regex ".+conf\/locale\/en$")
```

Comment:
ecommerce uses `tox` for all of its make commands. This will exclude `.tox` contents (instead of refactoring the Makefile)